DXRBalance/DeusEx/Classes/ShowerFaucet.uc

```diff
@@ -3,14 +3,22 @@ class ShowerFaucet injects ShowerFaucet;
 function Frob(actor Frobber, Inventory frobWith)
 {
 	local DeusExPlayer player;
+    local DXRando      dxr;
+    local bool         wasOnFire;
+
+	player = DeusExPlayer(Frobber);
+    wasOnFire = (player != None && player.bOnFire);
 
 	Super.Frob(Frobber, frobWith);
 
-	player = DeusExPlayer(Frobber);
-	if (player != None && player.bOnFire)
+	if (wasOnFire && !player.bOnFire)
```

Comment:
actually this is a little weird because if you're on fire and the shower is on, you have to double right click to extinguish yourself? maybe we should put it back and consider it QoL?